src/KeyManagement.sol

```diff
@@ -13,9 +13,9 @@ abstract contract KeyManagement is IKeyManagement, BaseAuthorization {
     using KeyLib for *;
     using SettingsLib for Settings;
 
-    EnumerableSetLib.Bytes32Set keyHashes;
-    mapping(bytes32 keyHash => bytes encodedKey) keyStorage;
-    mapping(bytes32 keyHash => Settings settings) keySettings;
+    EnumerableSetLib.Bytes32Set public keyHashes;
+    mapping(bytes32 keyHash => bytes encodedKey) public keyStorage;
+    mapping(bytes32 keyHash => Settings settings) public keySettings;
```

Comment:
I wonder if keySettings should be private? 

We kind of want people to use getKeySettings because keySettings with the root key hash has custom logic

Reply:
maybe actually same for keyStorage mapping

Reply:
oh good point